packages/store/tests/state.spec.ts

```diff
@@ -10,7 +11,7 @@ describe('Store', () => {
     @State({
       name: 'moo'
     })
-    class BarState {}
+    class BarState { }
```

Comment:
are you using the exact same prettier config that is set in the project? for example in my project it actually removes the space between those brackets.

Reply:
I haven't made any choice regarding prettier. I was relying on the default setup. Is there anything specific I need to do? I think it runs as part of the commit.

Reply:
It should yes maybe something is different in my local setup then